### PR TITLE
add patch request for edit products

### PR DESCRIPTION
**Pull request by rasoulreisi**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -48,6 +48,13 @@ server.use((req, res, next) => {
     next()
   }
 })
+server.use((req, res, next) => {
+  if (req.method === 'PATCH' && req.headers['content-type'] != 'application/json') {
```
Comment (unicornist) [inline server.js:52]: I was forgotten that I wrote this with an IF, please update the original middleware and add patch condition.

Comment (unicornist) [reply server.js:52]: and of course, remove this duplicate middleware.

Comment (unicornist) [reply server.js:52]: also update the comment above the middleware

Comment (rasoulreisi) [reply server.js:52]: Ok, I will solve this problem.

**Pull request merged**